Templates/CSharp/Desktop/MSTestDesktopCSharp.csproj

```diff
@@ -1,10 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project  Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TestTemplatesRoot Condition="$(TestTemplatesRoot) == ''">$(MSBuildThisFileDirectory)..\..\..\</TestTemplatesRoot>
   </PropertyGroup>
-  <Import Project="$(TestTemplatesRoot)scripts\build\TestTemplates.Settings.targets" />
+  <!--Import Project="$(TestTemplatesRoot)scripts\build\TestTemplates.Settings.targets" /-->
   <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <OutputPath Condition="$(OutputPath) == ''">$(TestTemplatesRoot)artifacts\$(Configuration)\$(MSBuildProjectName)\</OutputPath>
```

Comment:
I'd expect Arcade to define paths, this shouldn't be needed if you're using Arcade

Reply:
Yes, don't define these paths.